lib/Backend/GlobOpt.cpp

```diff
@@ -4348,11 +4359,6 @@ bool GlobOpt::CollectMemcopyStElementI(IR::Instr *instr, Loop *loop)
     SymID srcSymID = GetVarSymID(src1->GetStackSym());
 
     // Prepare the memcopyCandidate entry
-    if (loop->memOpInfo->candidates->Empty())
-    {
-        // There is no ldElem matching this stElem
-        return false;
-    }
     Loop::MemOpCandidate* previousCandidate = loop->memOpInfo->candidates->Head();
```

Comment:
ok, makes sense.

---
In reply to: [65933989](https://github.com/Microsoft/ChakraCore/pull/1015#discussion_r65933989) [](ancestors = 65933989)